src/main/java/uz/hiparts/hipartsuz/controller/OrderController.java

```diff
@@ -0,0 +1,102 @@
+package uz.hiparts.hipartsuz.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uz.hiparts.hipartsuz.dto.OrderDto;
+import uz.hiparts.hipartsuz.model.Order;
+import uz.hiparts.hipartsuz.model.User;
+import uz.hiparts.hipartsuz.model.enums.OrderType;
+import uz.hiparts.hipartsuz.model.enums.PaymentType;
+import uz.hiparts.hipartsuz.service.OrderService;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController("/api/v1/order")
+public class OrderController {
+    private final OrderService orderService;
+
+    @PostMapping("/create")
+    public void createOrder(@Valid Order order) {
+        orderService.create(order);
```

Comment:
After the order is created, the order information should be displayed to the user